custom_components/tahoma/cover.py

```diff
@@ -229,6 +229,32 @@ async def async_stop_cover_tilt(self, **_):
             self.select_command(COMMAND_STOP_IDENTIFY, COMMAND_STOP, COMMAND_MY)
         )
 
+    @property
+    def is_opening(self):
+        """Return if the cover is opening or not."""
+        return next(
+            (
+                True
+                for execution in self.coordinator.executions.values()
+                if execution.get("deviceurl") == self.device.deviceurl
+                and execution.get("command_name") in list(COMMAND_OPEN, COMMAND_UP)
```

Comment:
I suggest you to use this syntax: `execution["deviceurl"]`. We know the value will be always present, and this is shorter.